src/js/components/award/shared/awardAmountsSection/charts/AwardAmountsChart.jsx

```diff
@@ -92,7 +92,7 @@ const buildNormalProps = (awardType, data, hasFileC) => {
                         rawValue: data._fileCObligated,
                         denominatorValue: data._totalObligation,
                         value: data.fileCObligatedAbbreviated,
-                        text: 'COVID-19 Response Obligations Amount',
+                        text: 'COVID-19 Obligated Amount',
```

Comment:
are we sure we don't want spending obligated amount or something?

Reply:
This language has been cleared w Ross and Andrew - for consistency with the other labels in the chart and with the Advanced Search columns we went with "Obligated" and "Outlayed," and we figured that adding "Spending" in the label would be redundant

Reply:
When we removed "Response" from the Advanced Search columns we also kept "Spending" out of those headings as well